src/components/portal/lookup-injection-registration/VaccinationRegistrationTable.tsx

```diff
@@ -0,0 +1,120 @@
+'use client';
+
+import {
+  Chip,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  styled
+} from '@mui/material';
+import { VaccinationRegistration } from '@src/app/(mainPage)/portal/(accountPage)/lookup-injection-registration/page';
+import dayPhases from '@src/utils/constants/dayPhases';
+import dayjs from 'dayjs';
+import React, { FC } from 'react';
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(even)': {
+    backgroundColor: theme.palette.action.hover
+  },
+  '&:last-child td, &:last-child th': {
+    border: 0
+  }
+}));
+
+const TableHeadCell: FC<{ label: string }> = ({ label }) => {
+  return (
+    <TableCell
+      align="center"
+      sx={{
+        backgroundColor: (theme) => theme.palette.action.hover
+      }}>
+      <Typography variant="body1" fontWeight={500}>
+        {label}
+      </Typography>
+    </TableCell>
+  );
+};
+
+const TableBodyCell: FC<{ label: string | number }> = ({ label }) => {
+  return (
+    <TableCell align="center">
+      <Typography variant="body2">{label}</Typography>
+    </TableCell>
+  );
+};
+
+const registrationStatus = {
+  SUCCESS_REGISTRATION: 0,
+  ACCEPTED: 1
+};
+
+interface VaccinationRegistrationTableProps {
+  vaccineRegistrationList: VaccinationRegistration[];
```

Comment:
```suggestion
  vaccineRegistrations: VaccinationRegistration[];
```

chú ý cách đặt tên, ưu tiên số nhiều số ít, hạn chế dùng List